pkg/api/clustermanager.go

```diff
@@ -0,0 +1,153 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// SyncSetList represents a list of SyncSets for a given cluster.
+type SyncSetList struct {
+	SyncSets []*SyncSet `json:"value"`
+	// The link used to get the next page of operations.
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+type ClusterManagerConfigurationList struct {
```

Comment:
nit: missing doc comment
```suggestion

// ClusterManagerConfigurationList represents a list of ClusterManagerConfiguration for a given cluster.
type ClusterManagerConfigurationList struct {
```